tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -205,6 +205,9 @@
           ansiblePort: 22
           ansibleUser: root
           ansibleVars:
+            os_net_config_iface: {{ dataplane_os_net_config_iface | default ('nic1') }}
+            os_net_config_set_route: {{ dataplane_os_net_config_set_route | default(true) | bool }}
+            os_net_config_dns: {{ dataplane_os_net_config_dns | default("") }}
```

Comment:
sanity check for earlier conversation with @jistr about these values.

in the CI run here where we are not passing any of those 'dataplane_os_net_config_ ' vars the defaults are working as before:

https://logserver.rdoproject.org/00/200/c74c028d20b5831fa4bd0f92b90bf1721f900265/github-check/data-plane-adoption-github-rdo-centos-9-extracted-crc/54688aa/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2023-11-30T10:23:56EST.log

```
TASK [dataplane_adoption : deploy dataplane] ***********************************
...

ansibleVars:\n        os_net_config_iface: nic1\n        os_net_config_set_route: True\n        os_net_config_dns: \n        service_net_map

```

and tests complete successfully